### ISSUE TITLE
Allow to register "global" param providers

### ISSUE THREAD

**Issue by lolautruche** [lolautruche/EzCoreExtraBundle#38]
Global param providers would be called for all views, so that it won't be necessary to configure them for each and every views.

**Comment by flovntp** [comment]
Allow to use ParamProviders for twig_globals (like this, even if it's a custom route from SF, paramProvider would be useable)

**Comment by lolautruche** [comment]
@flovntp For `twig_globals` it will be difficult since param providers are bound to `ContentView`, and it won't work for regular symfony controllers.

**Comment by andrerom** [comment]
So this will inject variables for all content views then, kind of like a default set of params each and every override rule can overload? Will it be siteaccess aware?

**Comment by lolautruche** [comment]
@andrerom Yes, that's the idea. Not sure of the point making the providers themselves siteaccess aware, but parameters passed to them could be.
